io.github.chris2511.xca.yaml

```diff
@@ -35,3 +44,11 @@ modules:
         path: 0002-desktop-icon-fix.patch
     post-install:
       - install -Dm644 io.github.chris2511.xca.metainfo.xml /app/share/metainfo/io.github.chris2511.xca.metainfo.xml
+      - sed -i 's/Icon=xca;/Icon=io.github.chris2511.xca/g' /app/share/applications/xca.desktop
```

Comment:
> If renaming exported files to use the application ID is not possible, flatpak-builder allows them to be renamed as part of the build process. This can be done by specifying one of the following properties in the manifest:
> rename-icon - rename the application icon

https://docs.flatpak.org/en/latest/manifests.html#file-renaming